webcompat/static/js/lib/issues.js

```diff
@@ -53,6 +53,20 @@ md.linkify.add("#", {
     match.url = "/issues/" + match.url.replace(/^#/, "");
   }
 });
+
+var checkIfReportedURLOpeningA = function(tokens) {
+  if (
+    tokens[0].markup === "**" &&
+    tokens[1].content === "URL" &&
```

Comment:
actually, let's make this fail faster by checking the first 2 tokens as an initial check.